src/app/utils/accountPageUtils.js

```diff
@@ -13,19 +58,19 @@ export const makeRequest = (
   contentType,
   setIsLoading,
 ) => {
-  const url = `${appConfig.baseUrl}/api/account/${contentType}`;
+  const url = `${appConfig.baseUrl}/api/account /${contentType}`;
```

Comment:
Thanks for removing the space in the url. Here's another instance and there's one more on line 69. I think that's all of them.